include/amici/solver.h

```diff
@@ -384,6 +384,24 @@ class Solver {
      */
     void setAbsoluteToleranceQuadratures(double atol);
 
+    /**
+     * @brief returns the steady state simulation tolerance factor.
+     * Steady state simulation tolerances are the product of the simulation
```

Comment:
```suggestion
     * @brief returns the steady state simulation tolerance factor.
     *
     * Steady state simulation tolerances are the product of the simulation
```
Otherwise the full description is considered part of `brief`.